lib/mpl_toolkits/axes_grid1/axes_size.py

```diff
@@ -19,12 +19,27 @@ class _Base:
     def __rmul__(self, other):
         return Fraction(other, self)
 
+    def __mul__(self, other):
+        return Fraction(self, other)
+
+    def __div__(self, other):
+        return Fraction(self, 1/other)
```

Comment:
Similar to above `mysize / 2` is half of `mysize`. Here, it clearly does not make sense to divide something by a size. So this should not be added.